Sources/YBottomSheet/Enums/AccessibilityIdentifiers.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  AccessibilityIdentifiers.swift
+//  YBottomSheet
+//
+//  Created by Dev Karan on 25/04/23.
+//  Copyright © 2023 Y Media Labs. All rights reserved.
+//
+
+/// Accessibility Identifiers
+enum AccessibilityIdentifiers {
```

Comment:
Nest these within BottomSheetController and rename file to `BottomSheetController+ AccessibilityIdentifiers.swift`